test/topology/util.py

```diff
@@ -453,7 +453,7 @@ def get_uuid_from_str(string: str) -> str:
     return uuid
 
```

Comment:
Please make sure the commit message and the cover letter are the same. The commit message misses the paragraph about backporting and the cover letter misses Fixes and Refs.

For one-commit PRs, only the commit message remains, but the best (and the simplest) practice is to keep the commit message and the cover letter the same.